utils/formatting.js

```diff
@@ -0,0 +1,171 @@
+const posenet = require('@tensorflow-models/posenet');
+const {angle} = require('./scaling');
+const {createCanvas, loadImage} = require('canvas');
+
+const singlePoseNet = () => {
+  let poseNetConfig = {
+    algorithm: 'single-pose', //other option: multi-pose
+    input: {
+      architecture: 'MobileNetV1',
+      outputStride: 16,
+      inputResolution: {width: 640, height: 360},
+      multiplier: 0.75,
+      quantBytes: 2
+    },
+    singlePoseDetection: {
+      minPoseConfidence: 0.1,
+      minPartConfidence: 0.5
+    },
+    output: {
+      showVideo: true,
+      showPoints: true
+    }
+  };
+  return posenet
+    .load({
+      architecture: poseNetConfig.input.architecture,
+      outputStride: poseNetConfig.input.outputStride,
+      inputResolution: poseNetConfig.input.inputResolution,
+      multiplier: poseNetConfig.input.multiplier,
+      quantBytes: poseNetConfig.input.quantBytes
+    })
+    .then(net => net);
+};
+
+const getPose = async input => {
+  const net = await singlePoseNet();
+  return net.estimateSinglePose(input, {
+    flipHorizontal: true
+  });
```

Comment:
We might also have decodingMethod: 'single-person' as an option in the net.estimatePoses function